src/app/phase2/new-team-respond/new-team-response.component.ts

```diff
@@ -112,6 +135,20 @@ export class NewTeamResponseComponent implements OnInit {
     }));
   }
 
+  setSelectedLabelColor(labelValue: string, labelType: string) {
+    switch (labelType) {
+      case 'severity':
+        this.selectedSeverityColor = this.severityValues.filter(x => x.labelValue === labelValue)[0].labelColor;
+        break;
+      case 'type':
+        this.selectedTypeColor = this.issueTypeValues.filter(x => x.labelValue === labelValue)[0].labelColor;
+        break;
+      case 'responseTag':
+        this.selectedResponseColor = this.responseList.filter(x => x.labelValue === labelValue)[0].labelColor;
+        break;
+    }
+  }
```

Comment:
This line is being repeated. Can avoid?
`this.selectedSeverityColor = this.severityValues.filter(x => x.labelValue === labelValue)[0].labelColor;`